### PR TITLE
Added release items for PHP-FPM container

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

Added the following release items to the Docker 1.0.0 release notes:

- PHP-FPM container updates
    -  Node.js support (MAGECLOUD-3953)
    -  ionCube support (MAGECLOUD-4354)
- Docker command update:
   -  `expose db port to host` option for `ece-docker build:compose` command (MAGECLOUD-4544)

## Affected DevDocs pages

https://devdocs.magento.com/cloud/release-notes/mcd-release-notes.html

### CODE REVIEW

File: src/cloud/release-notes/mcd-release-notes.md [hunk 1]
```diff
@@ -36,17 +40,19 @@ The release notes include:
 
       -  {:.new}The base image for the [Magento Cloud Varnish container] is now based on the `centos` Docker image.
 
-   -  {:.new}**Improved default timeout configuration for the Magento Cloud TLS and Varnish containers**–[Fix submitted by Mathew Beane of Zilker Technologies](https://github.com/magento/magento-cloud-docker/pull/78)<!--MAGECLOUD-4460-->
+   -  {:.new}**Improved default timeout configuration for the Magento Cloud TLS and Varnish containers**–[Fix submitted by Mathew Beane from Zilker Technologies(https://github.com/magento/magento-cloud-docker/pull/78)<!--MAGECLOUD-4460-->
```
Comment (hguthrie) [inline src/cloud/release-notes/mcd-release-notes.md]:
```suggestion
   -  {:.new}**Improved default timeout configuration for the Magento Cloud TLS and Varnish containers**–[Fix submitted by Mathew Beane from Zilker Technologies](https://github.com/magento/magento-cloud-docker/pull/78)<!--MAGECLOUD-4460-->
```

File: src/cloud/release-notes/mcd-release-notes.md [hunk 2]
```diff
@@ -60,6 +66,10 @@ The release notes include:
 
    -  {:.fix}Renamed the `./bin/docker` file to `./bin/magento-docker` to fix an issue that caused some Docker environments to break because the `./bin/docker` file overwrites existing Docker binary files. This is a [backward incompatible change] that requires updates to your scripts and commands.<!-- MAGECLOUD-4038 -->
 
+   -  {:.new}**Added an option to expose the database port to the host**–[Updated submitted by Adarsh Manickam](https://github.com/magento/magento-cloud-docker/pull/101).
+
+      Use the `--expose-db-port=<PORT>` option to expose the database port to the host when building the `docker-compose.yml` file: `bin/ece-docker build:compose --expose-db-port=<PORT>`<!--MAGECLOUD-4454-->
```
Comment (hguthrie) [inline src/cloud/release-notes/mcd-release-notes.md]: Formatting here seems different than the rest of the items in the list. Seems like this line belongs before the "Update submitted by..."

File: src/cloud/release-notes/mcd-release-notes.md [hunk 3]
```diff
@@ -60,6 +66,10 @@ The release notes include:
 
    -  {:.fix}Renamed the `./bin/docker` file to `./bin/magento-docker` to fix an issue that caused some Docker environments to break because the `./bin/docker` file overwrites existing Docker binary files. This is a [backward incompatible change] that requires updates to your scripts and commands.<!-- MAGECLOUD-4038 -->
 
+   -  {:.new}**Added an option to expose the database port to the host**–[Updated submitted by Adarsh Manickam](https://github.com/magento/magento-cloud-docker/pull/101).
```
Comment (hguthrie) [inline src/cloud/release-notes/mcd-release-notes.md]:
```suggestion
   -  {:.new}**Added an option to expose the database port to the host**–[Fix submitted by Adarsh Manickam](https://github.com/magento/magento-cloud-docker/pull/101).
```

File: src/cloud/release-notes/mcd-release-notes.md (hunk 2 above)
Comment (meker12) [reply src/cloud/release-notes/mcd-release-notes.md]: was going back and forth on where to put the contributor acknowledgement-- I think I'll move them all to the end of the release note.

**Pull request merged**